### PR TITLE
Feature - Added example to docs for Portal

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Description
I have changed the background so it is easy to test the component. And want some suggestions on the placement. I have tried to make it the same as other libraries have. And the width of the portal is 0 by default, is it correct.

## Screenshots:
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/particles/portal/component.tsx [hunk 1]
```diff
@@ -25,31 +25,74 @@ const StyledPortal = styled.div<IStyledPortalProps>`
...
+export enum Placement {
+  bottomLeft = 'bottom-left',
+  bottomCenter = 'bottom-center',
+  bottomRight = 'bottom-right',
+  topLeft = 'top-left',
+  topCenter = 'top-center',
+  topRight = 'top-right'
+}
+
+export function getOffsetTop(rect: any, vertical: number | string): number {
```
Comment (MoiyadKayda) [inline src/particles/portal/component.tsx:37]: Please change the type of the `rect` argument.

Comment (MoiyadKayda) [reply src/particles/portal/component.tsx:37]: I was not able to get the type of this arg. The value of this arg is going to be of `anchorElementNodeRect = props.anchorElement?.current?.getBoundingClientRect();`

File: src/particles/portal/component.tsx [hunk 2]
```diff
@@ -25,31 +25,74 @@ const StyledPortal = styled.div<IStyledPortalProps>`
...
+  } else if (vertical === 'center') {
+    offset = rect.height / 2;
+  } else if (vertical === 'bottom') {
+    offset = rect.height;
+  }
+
+  return offset;
+}
+
+export function getOffsetLeft(rect: any, horizontal: number | string) {
```
Comment (MoiyadKayda) [inline src/particles/portal/component.tsx:50]: Same here

**Review by krishan711 — APPROVED** [review]

File: src/molecules/list/component.tsx [hunk 3]
```diff
@@ -46,7 +46,9 @@ interface IListProps extends IMoleculeProps<IListTheme>, IMultiChildProps<IListI
 
 export const List = (props: IListProps): React.ReactElement => {
```
Comment (krishan711) [inline src/molecules/list/component.tsx:47]: Please update changelog

**Review by krishan711 — CHANGES_REQUESTED** [review]
pls update changelog

**Review by krishan711 — APPROVED** [review]

**Pull request merged**